src/main/java/com/pingcap/tikv/RegionStoreClient.java

```diff
@@ -63,6 +65,95 @@ public ByteString get(ByteString key, long version) {
         return getHelper(resp);
     }
 
+    public void rawPut(ByteString key, ByteString value,Context context) {
+        RawPutRequest rawPutRequest = RawPutRequest.newBuilder()
+                .setContext(context)
+                .setKey(key)
+                .setValue(value)
+                .build();
+
+        // handle NotLeader
+        Function<RawPutResponse, Exception> errorHandler = x -> {
```

Comment:
Make it a separated class since it's likely other grpc call has similar logic. Except extracting error info part, all region error has same handling procedure.